app/views/editMyGroups.scala.html

```diff
@@ -0,0 +1,96 @@
+@import _root_.helper.MDLForms._
+@import models._
+@import java.util.UUID
+@import models.formModels.AddUserToGroupFormData
+@(currentUser: User, currentUserRights: Authorization.UserRights)(form: Form[AddUserToGroupFormData])(userGroups: List[UserGroup], users: List[User], applications: List[Application])(implicit webJarsUtil: org.webjars.play.WebJarsUtil, flash: Flash, messagesProvider: MessagesProvider, request: RequestHeader)
+
+@display(users: Seq[User], groupId: UUID)(header: Html)(footer: Html) = {
+    <div class="group-container mdl-cell mdl-cell--12-col mdl-shadow--2dp mdl-color--white" id="group-@groupId">
+        <div>
+            <div class="header">@header</div>
+        </div>
+        <table class="group-table mdl-data-table mdl-js-data-table">
+        @for(user <- users) {
+            <tr class="no-hover td--clear-border">
+                <td class="mdl-data-table__cell--non-numeric @if(user.disabled) { text--strikethrough }"><span class="application__name">@user.name</span>
+                    <i></i>
+                    <br>
+                    <span class="application__subject">@user.qualite</span></td>
+                <td class="mdl-data-table__cell--non-numeric @if(user.disabled) { text--strikethrough }">@user.email <br>
+                </td>
+                <td class="mdl-data-table__cell--non-numeric mdl-data-table__cell--content-size">
+                    @if(user.groupAdmin) {
+                        <span class="tag tag--responsable">Responsable</span>
+                    }
+                    @if(user.admin) {
+                        <span class="tag tag--admin">Admin</span>
+                    }
+                </td>
+                <td class="mdl-data-table__cell--non-numeric mdl-data-table__cell--content-size">
+                    @if(user.instructor) {
+                        <span class="tag tag--instructor">Instructeur</span>
+                    }
+                    @if(user.helper) {
+                        <span class="tag tag--aidant">Aidant</span>
+                    }
+                </td>
+                <td class="mdl-data-table__cell--non-numeric mdl-data-table__cell--content-size">
+                    <div id="answers-90500d58-4345-4538-a1d7-ff80e87b5398" class="vertical-align--middle">
+                        <i class="material-icons icon--light">chat_bubble</i>
+                        <span class="application__anwsers">@applications.count(_.creatorUserId == user.id) demandes</span>
+                        <br>
+                        <i class="material-icons icon--light">question_answer</i>
+                        <span class="application__anwsers">@applications.count(_.invitedUsers.contains(user.id))
+                            sollicitations</span>
+                    </div>
+                </td>
+                <td class="remove-link-panel">
+                @if(user.id != currentUser.id) {
```

Comment:
ah mince cats n'est pas importé ?
```suggestion
                @if(user.id =!= currentUser.id) {
```